src/components/scenes/WcConnectScene.tsx

```diff
@@ -75,6 +75,7 @@ export const WcConnectScene = (props: Props) => {
       navigation.navigate('wcConnections', {})
     } catch (error: any) {
       console.error(`WalletConnect connection error: ${error.message}`)
```

Comment:
I missed this in the last review, but `error.message` is forbidden. Do `String(error)`. Or just drop the whole line, since we already `showError` on the next line.

Reply:
I'll leave it for additional context and print appropriately.